src/styles/_loops.scss

```diff
@@ -0,0 +1,7 @@
+@for $month-day-count from 28 through 31 {
+  .calendar--month-length-#{$month-day-count} {
+    .calendar__day:nth-child(n + #{$month-day-count}) {
+      display: none;
+    }
+  }
+}
```

Comment:
this code belongs to calendar block. There's no need to create separate file for it.